src/views/Profile/components/EditProfileModal/RemovePage.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react'
```

Comment:
why this component is called `RemovePage` if it's a modal ?

Reply:
It is a "view" inside the modal. I see the confusion; I have renamed everything to view.